src/test/java/org/apache/sling/resourceresolver/impl/mapping/ResourceMapperImplTest.java

```diff
@@ -94,6 +95,7 @@ public ResourceMapperImplTest(boolean optimiseAliasResolution) {
     @Before
     public void prepare() throws LoginException {
 
+        ctx.registerService(ResourceResolverMetrics.class, Mockito.mock(ResourceResolverMetrics.class));
```

Comment:
I think we shouldn't mix Sling mocks and mockito here. Can you just pass the plain object?